src/Katanox/Http/GuzzleClient.php

```diff
@@ -44,7 +44,7 @@ public function request(
             }
 
             $headers['Authorization'] = sprintf('Bearer %s', $apiKey);
-            $headers['User-Agent'] = 'katanox-php/' . Version::get();
+            $headers['User-Agent'] = 'katanox-php/'.Version::get();
```

Comment:
cs-fixer unfortunately hasn't implemented this rule, so please put the space manually so that styleci passes